source/interfaces/q2studio.rst

```diff
@@ -7,6 +7,15 @@ QIIME 2 Studio (``q2studio``) is a graphical user interface for QIIME 2. This do
 
 .. note:: QIIME 2 Studio currently requires a source installation. In the future, we will provide a package distribution/installer which will greatly simplify the installation process presented here. An experimental build of QIIME 2 Studio is available as a pre-installed package in the :doc:`VirtualBox image <../install/virtual/virtualbox>`.
 
+Before being able to successfully run QIIME 2 Studio, you must always confirm that your QIIME 2 ``conda`` environment is activated. Failing to activate your environment prior to starting QIIME 2 Studio will cause issues and most likely crash the application. If you have not activated your ``conda`` environment already, you should do so now:
+
+.. command-block::
+	:no-exec:
+
+	source activate qiime2-2017.6
+
+If you run into an error at this point, then there is probably an issue with your install of QIIME 2. In that case, you should revisit the :doc:`install documentation <../install/index>` and make sure you have successfully installed QIIME 2 and all of its dependencies on your system.
```

Comment:
> Do you think we need to mention again that QIIME 2 must be installed in the environment, or have we covered that sufficiently?

If you included the edits suggested above, this has already been covered.

```
Please note, your ``conda`` environment might have a 
...
```

👍 🏅